.circleci/config.yml

```diff
@@ -0,0 +1,103 @@
+version: 2.1
+
+executors:
+  node_sdk:
+    docker:
+      - image: circleci/node:latest
+    working_directory: ~/repo
+
+jobs:
+  minor_release:
+    executor: node_sdk
+    steps:
+      - checkout
+      - build_and_release:
+          release_type: major
+  major_release:
+    executor: node_sdk
+    steps:
+      - checkout
+      - build_and_release:
+          release_type: major
+
+commands:
+  build_and_release:
+    parameters:
+      account_slug:
+        type: string
+        default: dreamdataio
+      release_type:
+        type: enum
+        enum: ['major', 'minor']
+      project_id:
+        type: string
+        default: api-dreamdata-cloud-production
+
+    steps:
+      - run:
+          name: Set semver
+          command: echo export SEMVER=$(eval node -p "require\(\'./package.json\'\).version") >> $BASH_ENV
+      - run:
+          name: Set major version
+          command: echo export MAJOR_VER=$SEMVER | cut -d. -f1 >> $BASH_ENV
+      - run:
+          name: Echo SEMVER
+          command: echo "Semver v${SEMVER}"
+      - run:
+          name: Get latest LSB release
+          command: sudo apt-get -y update && sudo apt-get -y install lsb-release
+      - run:
+          name: Install Google Cloud SDK
+          command: export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" && echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add - && sudo apt-get update -y && sudo apt-get install google-cloud-sdk -y
```

Comment:
Can we somehow wrap this line? It's hard to read this 😝

Reply:
wrapped